src/main.ts

```diff
@@ -120,7 +120,8 @@ actionsToolkit.run(
     const buildCmd = await toolkit.buildx.getCommand(args);
     const buildEnv = Object.assign({}, process.env, {
       BUILDX_BAKE_GIT_AUTH_TOKEN: gitAuthToken,
-      BUILDX_METADATA_WARNINGS: 'true'
+      BUILDX_METADATA_WARNINGS: 'true',
+      BAKE_ALLOW_REMOTE_FS_ACCESS: 'true'
```

Comment:
I'm using this env atm https://github.com/docker/buildx/blob/17eff25fe591ef7ecf0a7046cb55f847db42f575/bake/entitlements.go#L270 but maybe an override would be better with `--set *.allow=fs=/` ?